ChangeLog.md

```diff
@@ -1,11 +1,105 @@
 
 # Change Log
 
-## Version 10.XX.XX
+## Version 10.3.0
 
-### Breaking change
+### Breaking changes
+
+1. (TBC based on outcome of backslash escape vs --strip-top-dir)  The `*` character is no longer supported as a wildcard in URLs.  (It
+   remains supported in local file paths.)
+   1. The one execption is that `/*` is still allowed at the very end of the "path" section of a
+      URL. This is illustrated by the difference between these two source URLs:
+      `https://account/container/virtual?SAS` and 
+      `https://account/container/virtualDir/*?SAS`.  The former copies the virtual directory
+      `virtualDir` by creating a folder of that name at the destination.  The latter copies the
+      _contents_ of `virtual` dir directly into the target without creating a folder named "virtualDir".
+   1. If you need to refer a literal `*` in the name of a blob, e.g. for a blob named "*", escape it with a backslash as `\*`
+1. The `--include` and `--exclude` parameters have been replaced by `--include-pattern` and
+   `--exclude-pattern` (for filenames) and `--include-path` and `--exclude-path` (for paths,
+   including directory and filenames).
+   The new parameters have behaviour that is better defined in complex situations (such as
+   recursion).  The `*` wildcard is supported in the pattern parameters, but _not_ in the path ones.
+1. There have been two breaking changes to the JSON output that is produced if you request
+   JSON-formatted output. The `sync` command's output in JSON has changed for consistency reasons,
+   and the final message type, for `copy` and `sync` has changed its name from `Exit` to `EndOfJob`.
+   Tools using the JSON output format to integrate AzCopy should be aware.
+1. If downloading to "null" on Windows the target must now be named "NUL", according to standard
+   Windows conventions.  "/dev/null" remains correct on Linux. (This feature can be used to test
+   throughput or check MD5s without saving the downloaded data.) 
+1. The file format of the (still undocmented) `--list-of-files` parameter is changed.  (It remains
+   undocmented because, for simplicity, users are
+   encouraged to use the new `--include-pattern` and `--include-path` parameters instead.)
+
+### New features
+
+1. `sync` is supported from Blob Storage to Blob Storage, and from Azure Files to Azure Files.
+1. `copy` is supported from Azure Files to Azure Files, and (TBC?) from Blob Storage to Azure Files.
+1. Percent complete is displayed as each job runs.
+1. VHD files are auto-detected as page blobs.
+1. A new benchmark mode allows quick and easy performance benchmarking of your network connection to
+   Blob Storage. Run AzCopy with the paramaters `bench --help` for details.  This feature is in
+   Preview status.
+1. The location for AzCopy's "plan" files can be specified with the environment variable
+   `AZCOPY_JOB_PLAN_LOCATION`.
```

Comment:
Perhaps note that we still don't support moving the entire app folder-- our oAuth token is still stored in ~/.azcopy

Reply:
Done. THanks. BTW, that's only true on Windows. We use OS keyring on other OS's, right?  I hope that's right, because that's how I just worded it ;-)